presto-native-execution/CMakeLists.txt

```diff
@@ -153,6 +157,12 @@ set(FOLLY_WITH_DEPENDENCIES
     ${ZSTD}
     ${ZLIB_LIBRARIES})
 
+if(DEFINED FOLLY_BENCHMARK_STATIC_LIB)
```

Comment:
Is this change related to this PR?

Reply:
Yes, it was related to finding the folly library that has the benchmark infrastructure. It'll get removed along with the benchmark program.